jina/importer.py

```diff
@@ -112,9 +112,9 @@ def _path_import(absolute_path: str):
                 f'''
             {user_module_name} shadows one of built-in Python module name.
             It is imported as `{default_spec_name}.{user_module_name}`
-            
+
             Affects:
-            - Either, change your code from using `from {user_module_name} import ...` 
+            - Either, change your code from using `from {user_module_name} import ...`
```

Comment:
out of the scope of this PR, but im not really sure this warning is still valid @tadejsv